no-computer.md

```diff
@@ -1,69 +1,170 @@
 ### Domain
+
 _Describe the project's domain in five words._
 
 
+
+Design and creation of buildings?
+
+
+
 ### Interface (syntax)
+
 _How might the user interact with the language? What does programming look 
+
 like? Why is this the right way to interact with the problem domain?_ 
 
 
+
+A "program" would be the finished blueprint, so to interact with the language would be to draw the building on grid paper with specifications of length and type of materials. 
+
+
+
 ### Operation (semantics)
+
 _What might happen when a program runs? How does a program interact with the
+
 user? What kinds of errors might occur, and how might they be communicated to
+
 the user?_
 
 
+
+When a program "runs," people execute the building plans, put together the pieces specified in the blueprint, and voila, a building is made. Some errors would be buying the wrong materal, or having the wrong size for any piece. The construction workers would figure this out while trying to use the material/piece. More abstractly, there may be an error in physics calculations, and the building would topple.
+
+
+
 ### Expressiveness
+
 _What should be easy to do in this language? What should be possible, but
+
 difficult? What should be impossible or very difficult?_
 
 
+
+It should be easy to design a building. It's technically possible to make blueprints of nearly anything, but this depends on how lenient the definition of "building" is and where the line between "building blueprints" and "general-purpose blueprints" are.
+
+It should be impossible to do... anything outside of designing/planning the creation of physical objects.
+
+
+
 ### Related work
+
 _Are there any other DSLs in this domain? If not, describe how you know there
+
 aren't and conjecture why not. If so, describe them and provide links. How well 
+
 do they address the need? Are there any particularly admirable qualities of the
+
 language? Are there parts of the language you think could be improved?_
 
 
+
+Considering how open "building blueprints" could be taken, most likely not. Verbal planning may be considered a DSL, or a textual representation of a plan, though images work the best.
+
+
+
 ## The Project
+
 This section examines whether the idea makes for a good CS 111 project.
 
 
+
+It doesn't, at least not for pencil-and-paper blueprints.
+
+There are many ways to create an application for a building blueprint and then create a 3D model of it, but this has been done many times.
+
+
+
 ### Suitability
+
 _If someone were to work on this project, what percentage of their time would be
+
 spent directly engaging in the **language** aspects of this project (e.g.,
+
 making language design decisions), as opposed to "systems" aspects of the
+
 project (e.g., implementing a complicated semantics that doesn't require a lot
+
 of language design)?_
 
 
+
+That depends on what is implemented. The interpretation of the program isn't being done by a computer, so how much control would someone have over this?
```

Comment:
This is a good question. Perhaps one could render a textual description as a 3d model?
There would have to be an execution engine for that.